alexkates/hashnode-next
# Feature/draft preview

## What does this PR do?

Fixes #9

## Type of change




## How should this be tested?

- Create a draft in the Hashnode editor.
- Open the preview of this draft.
- The draft should be shown on your website.

### Things to remember

- [X] Ran `pnpm format`
- [X] Ran `pnpm lint`
